prince: version bump to v0.4

gabri94 opened the pull request:
Release of the new version of prince. The building method has been changed to CMake.

Signed-off-by: Gabriele Gemmi <gabriel@autistici.org>

zioproto commented on prince/files/prince.init:
```diff
@@ -0,0 +1,20 @@
+#!/bin/sh /etc/rc.common
+# Init script for Prince
+# Copyright (C) 2007 OpenWrt.org
```
Please update to 2018 the Copyright. Also check if there is a proper Licence header for files committed into OpenWrt.

gabri94 replied:
Corrected

zioproto commented on prince/Makefile:
```diff
@@ -0,0 +1,59 @@
+PKG_SOURCE:=prince-$(PKG_VERSION).tar.xz
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_SUBDIR:=prince-$(PKG_VERSION)
+PKG_SOURCE_URL:=https://github.com/AdvancedNetworkingSystems/poprouting.git
```
the package is called prince but the source repository is still called pop routing ?

gabri94 replied:
Thanks, I uniformed everything to prince.

zioproto commented on prince/Makefile:
```diff
@@ -0,0 +1,59 @@
+	SUBMENU :=Routing and Redirection
+	Mantainer :=Gabriele Gemmi <gabriel@autistici.org>
+	TITLE :=PopRouting daemon
+	URL :=https://github.com/gabri94/poprouting
```
This is not the same URL you are downloading the source code from

gabri94 replied:
I forgot to change it when we moved the repo.
Fixed it.

zioproto commented on prince/Makefile:
```diff
@@ -0,0 +1,59 @@
+	CATEGORY :=Network
+	SUBMENU :=Routing and Redirection
+	Mantainer :=Gabriele Gemmi <gabriel@autistici.org>
+	TITLE :=PopRouting daemon
```
PopRouting or prince ?

gabri94 replied:
prince

zioproto commented on prince/Makefile:
```diff
@@ -0,0 +1,59 @@
+	Prince is an open source implementation of the PopRouting Algorithm.
+	It has been developed as a Google Summer of Code Project in collaboration with Freifunk and the University of Trento.
+
+	Prince fetches topology data from a Link State routing daemon(OONF, OLSR, OSPF, etc),
```
Missing space before "("

gabri94 replied:
done

gabri94 commented:
Thanks for the review, I fixed all the errors, however I have  doubt:
Since I modified the make file, I increased the release number to '2'. Is it correct?

zioproto commented:
If you never distributed that version of the Makefile you should keep it to version number 1

zioproto commented on prince/Makefile:
```diff
@@ -0,0 +1,66 @@
+
+PKG_NAME:=prince
+PKG_VERSION:=v0.4
+PKG_RELEASE:=2
```
I think RELEASE 1 here is okay because you never distributed the other version, it was just in the code review

zioproto commented on prince/Makefile:
```diff
@@ -0,0 +1,66 @@
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/build/prince $(1)/usr/bin/prince
+	$(INSTALL_BIN) ./files/prince.init            $(1)/etc/init.d/prince
```
Why on line 57 and 58 you have these long spaces ? If there is not reason keep it to 1 space I guess

zioproto commented on prince/files/prince.init:
```diff
@@ -0,0 +1,20 @@
+
+start_service() {
+	procd_open_instance
+        procd_set_param command $PROG $CONF
```
There is an extra space here

gabri94 commented:
Fixed everything
